Review getting started document

Preeticp opened the pull request:
## What is the purpose of this change? What does it change?
Reviewed and modified getting started document.

Preeticp commented:
@cdrage request you to review the PR please.

cdrage commented on docs/getting-started.md:
```diff
@@ -1,281 +1,141 @@
 
-Requirements:
-  - `minishift` version 1.22+
+ 1. Start an OpenShift cluster, version 3.10.0 and above
```
i think `3.10` and above is fine

cdrage commented on docs/getting-started.md:
```diff
@@ -1,281 +1,141 @@
-If you are using `minishift` you'll need to start an OpenShift cluster with version 3.10.0+ and Service Catalog explicitly enabled.
+3. After you enable or start `minishift` use:
+ * `odo catalog list services` to list the services
+  * `odo service <verb> <servicename>` to list service catalog related operations
```
odd spacing here...

cdrage commented on docs/getting-started.md:
```diff
@@ -1,281 +1,141 @@
-After you've enabled / started `minishift`, you'll be able to list the services via `odo catalog list services` and service catalog related operations via `odo service <verb> <servicename>`.
-
-## Adding a Custom Builder
+### Adding a Custom Builder
```
Should Custom Builder be capitalized?

cdrage commented on docs/getting-started.md:
```diff
@@ -1,281 +1,141 @@
+### Adding a Custom Builder
 
-**This section assumes that the `oc` binary has been [installed](https://docs.openshift.org/latest/cli_reference/get_started_cli.html#installing-the-cli) and is present on the $PATH**
+OpenShift enables you to add a [custom image](https://docs.openshift.com/container-platform/3.7/creating_images/custom.html) to bridge the gap in the creation of custom images. A custom builder image usually includes the base image of [openshift/origin-custom-docker-builder](https://hub.docker.com/r/openshift/origin-custom-docker-builder/).
```
bridge the gap between the creation

cdrage commented on docs/getting-started.md:
```diff
@@ -1,281 +1,141 @@
 
-A custom builder image usually includes the base image of [openshift/origin-custom-docker-builder](https://hub.docker.com/r/openshift/origin-custom-docker-builder/).
+**Prerequisites:**
+oc binary is [installed](https://docs.openshift.org/latest/cli_reference/get_started_cli.html#installing-the-cli) and present on the `$PATH`
```
`oc`

Preeticp commented:
@cdrage I have addressed all your comments except the TOC part. I would suggest let's revisit this after we have done a round of review of all docs, so that we can have a consistent feel to the whole doc set. WDYT?
Please merge if you feel it's alright, I will proceed with the Installation doc.

amitkrout commented:
@Preeticp Some prow fixes went in, so can debase please.

Preeticp commented:
@amitkrout I have rebased, fixed the merge conflict. Reviewed and modified the new section which created the merge conflict. 
Please merge.

amitkrout commented:
/lgtm
